source/amend_project/configuration/index.html.md.erb

```diff
@@ -7,4 +7,229 @@ review_in: 1 day
 
 # Configuration options
 
-<%= gem_docs 'configuration.md' %>
```

Comment:
Does this mean we need to do a pull request on the gem repo to remove content?

Reply:
> This is the PR to add signposting for the content move: alphagov/tech-docs-gem#151

Reply:
Yup - I've raised this PR to cover the content move: https://github.com/alphagov/tech-docs-gem/pull/151